### PR TITLE
Document how update the state of the 'debugged' application

### PR DESCRIPTION
**Pull request by dalgassimou**

### CODE REVIEW

**Comment by dalgassimou** [comment]
link to download the latest vee debugger proxy: https://nextcloud.cross/index.php/s/4RH7NEG9Gp39x97

File: ApplicationDeveloperGuide/debuggerProxy.rst [hunk 1]
```diff
@@ -221,6 +221,13 @@ Open a shell terminal on your workstation and run the following command:
 
 Open the SDK and run a :ref:`Remote Java Application Launch <debug_on_device>` to debug your code.
 
+
+Refresh the state of the memory dump
```
Comment (agazardMicroeJ) [inline ApplicationDeveloperGuide/debuggerProxy.rst:225]: move just before the `VEE Debugger Proxy Options Summary` section

File: ApplicationDeveloperGuide/debuggerProxy.rst [hunk 2]
```diff
@@ -221,6 +221,13 @@ Open a shell terminal on your workstation and run the following command:
...
+------------------------------------
+
+While the VEE Debugger Proxy is running, you can continue to run the application and perform a new memory dump at a later time. With this new memory dump there's no need to stop and restart the VEE Debugger Proxy to use it, just stop the :ref:`Remote Java Application Launch <debug_on_device>`
```
Comment (agazardMicroeJ) [inline ApplicationDeveloperGuide/debuggerProxy.rst:228]: rewording proposal: If you want to debug your application using a new memory dump, you don't need to stop and restart the VEE Debugger Proxy. Simply overwrite the old memory dump files with the new ones, then restart the  :ref:`Remote Java Application Launch <debug_on_device>`.

**Review by frivieremicroej — CHANGES_REQUESTED** [review]
@agazardMicroeJ before merging this branche, the new proxy must first be published in next SDK 6 release, and download link updated to this new version

**Comment by rlouedec-microej** [comment]
I've just noticed that the jdwp-server 1.0.3 was published: https://forge.microej.com/ui/repos/tree/General/microej-sdk6-repository-release/com/microej/tool/jdwp-server/1.0.3

_Note that it does not publish any CHANGELOG._

Can we finalize this PR?

**Comment by Jetbl** [comment]
Download link updated, PR ready to be merged

**Comment by dalgassimou** [comment]
lgtm

**Review by rlouedec-microej — APPROVED** [review]
LGTM, thank you!

**Review by frivieremicroej — APPROVED** [review]

**Pull request merged**